docs/CONFIG.md

```diff
@@ -38,30 +38,23 @@ Configuration lives in [config.yml](../config.yml).
   - `max_concurrent_access_control_syncs`: The maximum number of concurrent access control syncs. Defaults to 1.
   - `job_cleanup_interval`: The interval (in seconds) to run job cleanup task.
   - `log_level`: Connector service log level. Defaults to `INFO`.
-- `native_service_types`: An array of supported native connectors (in service type).
```

Comment:
So `native_service_types` is supposed to be used only on Cloud, and that's why it's removed. Do you think we should remove `username` and `password` as well? It's more intuitive to use the API key for on-prem connector service.

Reply:
Fair enough. username/password is provided as an additional auth method, while `native_service_types` is something not supposed to be used by customers.